shared/schemas/search-widget-data-params.schema.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from 'zod';
+import {
+  VALID_SEARCH_WIDGET_DATA_OPERATORS,
+  WidgetDataFilterOperatorType,
+} from '@shared/dto/global/search-widget-data-params';
+
+export const SearchWidgetDataParamsSchema = z.object({
+  filters: z
+    .array(
+      z.object({
+        name: z.string(),
+        operator: z.enum(
+          VALID_SEARCH_WIDGET_DATA_OPERATORS as [
+            WidgetDataFilterOperatorType,
+            ...WidgetDataFilterOperatorType[],
```

Comment:
Can we simplify this? It's over-complicated

Reply:
@alexeh You mean like create the filter object and just having an array of the filter z.object?

Reply:
Not totally sure, but maybe use just a enum and avoid multiple level type casting. Just to simplify the code